Sprite.java

```diff
@@ -56,6 +57,7 @@ public Sprite(JSONObject jsonObj)
         costumeCount = FileUtils.getJSONArrayAttribute(this.jsonObj, 
```

Comment:
'this' keyword is not necessary here - since we're only referring to a single globalVariables field, just 'globalVariables' is sufficient. (my bad!)